docs/content/querying/filters.md

```diff
@@ -81,6 +91,14 @@ The following matches any dimension values for the dimension `name` between `'ba
 }
 ```
 
+The following matches values of the given two dimensions `dim1` and `dim2` are the same
```

Comment:
`The following matches rows where the values for the given two dimensions dim1 and dim2 are the same` would be more grammatically correct. Also I think this requires the row flag to be true, no?